contracts/OracleManagerApp.sol

```diff
@@ -15,25 +15,22 @@ contract OracleManagerApp is AragonApp {
   ITidbitDataFeedOracle public medianDataFeed;  // tidbit MedianDataFeedOracle to record median data throughout time
   uint public approvedDataFeedsLength;   // number of approvedDataFeeds
   mapping(address => bool) public approvedDataFeeds; // dataFeeds approved to be medianized
-  mapping(address => bool) dataFeedAlreadyRecorded; // transitory data structure useful only during function call recordDataMedian
```

Comment:
I still like my wording better. more specific takes less seconds to understand when going through the code

Reply:
can we call it `dataFeedIsSet`?